pkg/services/publicdashboards/database/database.go

```diff
@@ -37,6 +37,27 @@ func ProvideStore(sqlStore *sqlstore.SQLStore) *PublicDashboardStoreImpl {
 	}
 }
 
+// Gets list of public dashboards by orgId
+func (d *PublicDashboardStoreImpl) ListPublicDashboards(ctx context.Context, orgId int64) ([]PublicDashboardListResponse, error) {
+	resp := make([]PublicDashboardListResponse, 0)
+
+	err := d.sqlStore.WithTransactionalDbSession(ctx, func(sess *sqlstore.DBSession) error {
+		sess.Table("dashboard_public").
+			Join("LEFT", "dashboard", "dashboard.uid = dashboard_public.dashboard_uid AND dashboard.org_id = dashboard_public.org_id").
```

Comment:
I think this should be an `INNER` join. Do we want to list public dashboards not associated with a valid dashboard?

Reply:
That's a good point! As of right now, we don't have a way of seeing or deleting public dashboards not associated with a dashboard. I'm going to leave this here and make a ticket for adding a delete button to the table.

Reply:
I think we need both. A way to delete a valid public dashboard but also, should delete the public dashboard record whenever a dashboard is deleted. I don't see the point of showing an orphan public dashboard in this table. This is how it looks now with this type of join: 
![image](https://user-images.githubusercontent.com/20256983/195197667-79c010f5-331d-4339-8ad4-ecc8da450a8d.png)

Reply:
Yeah good catch Eze. I didn't think of the case where the original dashboard is deleted.

Reply:
@evictorero At first glance, this is simple, however, we have a mutual dependency between dashboards and publicdashboards and no message bus to handle communication between the two. This means that dashboards needs a publicdashboards service with a delete method and then we need to handle that failure state in dashboard code- I agree that it is the way it should be done, however, we received comments (marked resolved) from regarding coupling. I believe this is a larger discussion that has not be resolved yet.

I will make a ticket for handling that and solicit discussion from Ryan McKinley and Torkel

Reply:
https://github.com/grafana/grafana-partnerships-team/issues/421